src/policies/combine_policies_over_periods.py

```diff
@@ -369,19 +388,19 @@ def get_october_to_christmas_policies(
 
     """
     dates = pd.date_range("2020-10-01", "2020-12-23")
-    if work_multiplier is None:
+    if attend_work_multiplier is None:
         work_multiplier_path = BLD / "policies" / "work_multiplier.csv"
-        work_multiplier = pd.read_csv(work_multiplier_path, parse_dates=["date"])
-        work_multiplier = work_multiplier.set_index("date")
-        work_multiplier = work_multiplier.loc[dates]
+        attend_work_multiplier = pd.read_csv(work_multiplier_path, parse_dates=["date"])
```

Comment:
why does this not use the `get_attend_work_multiplier` function defined below?

Reply:
`get_attend_work_multiplier` is only used in `get_enacted_policies_of_2021` because the handling is different and much simpler in the past (here) than when extrapolating into the future.